modular_skyrat/modules/loadouts/loadout_items/loadout_datum_suit.dm

```diff
@@ -356,7 +356,7 @@ GLOBAL_LIST_INIT(loadout_exosuits, generate_loadout_items(/datum/loadout_item/su
 	item_path = /obj/item/clothing/suit/fallsparka
 
 /datum/loadout_item/suit/poly
-	name = "Polychromic Wintercoat"
+	name = "Colourable Wintercoat"
```

Comment:
Best call it recolorable to fit with the other gags loadout items

Also maybe needs to be repathed from /poly but that might mean people have to readd it to loadout

Either way good riddance to poly

Reply:
Also please, `Colorable` not `Colourable`.